### DISCUSSION TITLE
Pending apply status

### DISCUSSION THREAD

**Discussion by s7thamon** [jaredhendrickson13/pfsense-api#426] [Q&A]
Is there anywhere in the API to check if changes have been made to the firewall and need to be applied?

I see the route to apply firewall changes but not a status

**Comment by jaredhendrickson13** [comment]
Currently there is not an endpoint to read the apply status for firewall changes. Feel free to open a [feature request](https://github.com/jaredhendrickson13/pfsense-api/issues/new?assignees=&labels=backlog%2C+feature+request&projects=&template=feature-request.md&title=). Should be fairly simple to implement and could be included in a patch.

Thanks!
